tests/unit/test_db_connector.py

```diff
@@ -35,7 +38,22 @@ def test_get_by_id_mock_as_context_manager(mocker):
     mocker.patch("redis.Redis.get").return_value = json.dumps(object_to_save)
 
     result = None
-    with DBConnector() as db_connector:
+    with DBConnectorRedis.instance() as db_connector:
         db_connector.save(id_test, object_to_save)
         result = db_connector.get_by_id(id_test)
     assert result
```

Comment:
here it was missing the comparison, which I forgot to upload fyi

Reply:
comparison added